test/verify/check-realms

```diff
@@ -781,7 +781,20 @@ class TestPackageInstall(packagelib.PackageCase):
         if self.machine.image in ["rhel-8-1-distropkg"]:
             self.domain_sel = "#system-info-domain a"
         else:
-            self.domain_sel = "#system-info-domain button"
+            self.domain_sel = "#system_information_domain_button"
+
+    def checkTooltip(self, text):
+        b = self.browser
+        if self.machine.image in ["rhel-8-1-distropkg"]:
+            b.wait_present("#system-info-domain [data-original-title]")
+            b.mouse("#system-info-domain span", "mouseover")
+            b.wait_in_text(".tooltip-inner", text)
+            b.mouse("#system-info-domain span", "mouseout")
+        else:
+            b.wait_present("#system_information_domain_tooltip[data-test-has-tooltip]")
```

Comment:
This is the counterpart to the attribute definition above. What does that line do? AFAICS, just dropping it should have exactly the same result -- it waits  for `system_information_domain_tooltip`, hovers it, and then waits for the actual tooltip to pop up and checks its text. Why is that not sufficient?

Reply:
The `#system_information_domain_tooltip` is always present, even if there is no tooltip.  We might indeed change so that it only appears when there is a tooltip.  (Fingers crossed that this doesn't screw with the layout.)

Reply:
Right, that's the "hook" for attaching a notification to, and we need it for the hovering. It's fine to always be present. My question was, why do we need to wait for this attribute in the first place? It should be equivalent to waiting for `div.tooltip` to appear.

Reply:
Yes, removing the `<span>` unfortunately messes with the hostname change button.  So what about leaving the `<span>` in but only giving it the `id` when it actually has a tooltip?  I have done that.

Reply:
Right, you need the span so that the tooltip gets attached to a non-disabled element; tooltips on disabled elements don't work. This still looks funny, but I think I understand the reason now. Thanks!